Desktop/components/JASP/Widgets/FileMenu/PrefsUI.qml

```diff
@@ -190,17 +190,46 @@ ScrollView
 				startValue: 				languageModel.currentLanguage
 				onValueChanged: 			languageModel.currentLanguage = value
 
-				KeyNavigation.tab: 			altnavcheckbox
+				KeyNavigation.tab: 			useAlternativeLocale
 				
 			}
 			
 			CheckBox
 			{
-				id:							useUsLocale
-				label:						qsTr("Use American English locale for numbers")
-				checked:					languageModel.useUsLocale
-				onCheckedChanged:			languageModel.useUsLocale = checked
-				toolTip:					qsTr("Use US english locale for display of numbers, dates, times and currency. Decimal separator will be '.' for instance.")
+				id:							useAlternativeLocale
+				label:						qsTr("Use an alternative locale for numbers")
+				checked:					languageModel.useAlternativeLocale
+				onCheckedChanged:			languageModel.useAlternativeLocale = checked
+				toolTip:					qsTr("Use the locale specified below for display of numbers and currency.") //dates and times will have to be added later
+				
+				KeyNavigation.tab:			alternativeLocaleTerritory
+			}
+			
+			Group
+			{
+				enabled:					languageModel.useAlternativeLocale
+				DropDown
+				{
+					id:							alternativeLocaleLanguage
+					label:						qsTr("Alt. language")
+					values:		 				languageModel.altLanguages
+					startValue:				 	languageModel.currentAltLanguage
+					onValueChanged:				if(value != "") languageModel.currentAltLanguage = value
+					addEmptyValue:				false
+					KeyNavigation.tab:			alternativeLocaleTerritory
+				}
+				
+				DropDown
+				{
+					id:							alternativeLocaleTerritory
+					label:						qsTr("Alt. territory")
+					values:		 				languageModel.altTerritories
+					startValue:				 	languageModel.currentAltTerritory
+					onValueChanged:				if(value != "") languageModel.currentAltTerritory = value
+					addEmptyValue:				false
+					KeyNavigation.tab:			altnavcheckbox
+				}
+				
```

Comment:
Maybe if we use something like `Number(-12345.37).toLocaleString(Qt.locale(<local code>))` to have a preview following locale changed in the right side, will help users see expected changes visually.everything else seems fine to me, except the thousandths side effect (I'm still not used to it)😜

Reply:
> I can think of a situation where the user changes the locale but doesn't want to change the number representation

But then what do they chance the locale for?

And Im also still getting used to the thousands separators yeah :p

Reply:
- From now: change language -> number format changed also, then if you don't want to you should Specifying an Alternative locale
- What I mean is: the format does not follow the language changes, but follows the alternative locale.

I'm not particularly adamant about this though, since I can also change the language by changing the locale afterwards, it's just a matter of order. It would be nice to have some documentation though.

Reply:
We could turn the "alternative" on by default and then have the american thing be default.
The preview could be nice too and I guess I should indeed write some documentation.

I figured the tooltip would be enough :p